test/e2e/framework/kubernetes/get-external-crd.go

```diff
@@ -0,0 +1,65 @@
+package kubernetes
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+func downloadExternalCRDs(chartPath string) error {
+	crdUrls := []string{
+		"https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml",
+	}
+
+	for _, crdUrl := range crdUrls {
+		crd, err := fetchYAML(crdUrl)
+		if err != nil {
+			return err
+		}
+		if crd != nil {
```

Comment:
no need for this check as `fetchYAML` returns error or a byte slice, so if we get to this point we have a byte slice since the error would have been caught in the previous block.